src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java

```diff
@@ -1882,6 +1882,31 @@ public void testContainerWithMoreCpuOptions() throws Exception {
   }
 
   @Test
+  public void testContainerWithNanoCpusOptions() throws Exception {
+    requireDockerApiVersionAtLeast("1.19", "Container creation with NanoCpus option");
```

Comment:
Looking at the test failures, for instance, job [#1530.4](https://travis-ci.org/spotify/docker-client/jobs/241179322), I am guessing that you should require at least API version `1.25` for this test.